deployment/terraform/assets/cluster.tf

```
@@ -593,7 +593,7 @@ resource "aws_security_group" "redis" {
     security_groups = [aws_security_group.app[0].id, aws_security_group.metrics[0].id]
```

It seems `count` is resolved after the check on the number of elements in tuples is done, so this PR still errors to me with "Invalid Index" error, stating that `app_security_group.app is empty tuple`. I found that we can use `count.index` to bypass this:

```suggestion
    security_groups = [aws_security_group.app[count.index].id, aws_security_group.metrics[count.index].id]
```